README.md

```diff
@@ -1,17 +1,16 @@
-# {{ organization_name }} Analytics
+# dbt Training - Rapid Onboarding Exemplar
```

Comment:
Any reason why this template changed? This is the template for the RO and we have the client copy this / add it to their own repo and modify it

Reply:
I was updating to have an example of what a populated template would look like. Happy to revert if there's a strong opinion for keeping it the template, but the RO lesson plan [links to your gist](https://gist.github.com/christineberger/2d00a2f846b1b49359fde5c2f8f354fe), not the example project repo.

Reply:
I think it is confusing to both be an example and be a template for building your own.
I would opt for a filled out README and separately a template, either in this repo or another repo.

Reply:
+1 To Pat's idea